utils/highlightjs/highlightjs_carbon_lang.js

```diff
@@ -140,7 +140,7 @@ export default function (hljs) {
     scope: 'number',
     variants: [
       { match: /[1-9][_0-9]*(\.[_0-9]+(e[-+]?[1-9][0-9]*)?)?/ },
-      { match: /0x[_0-9A-F]+(\.[_0-9A-F]+(p[-+]?[1-9][0-9]*)?)?/ },
+      { match: /0x[_0-9a-fA-F]+(\.[_0-9a-fA-F]+(p[-+]?[1-9][0-9]*)?)?/ },
```

Comment:
The regex was correct; we're looking at requiring uppercase for consistency/readability. References:

https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/lexical_conventions/numeric_literals.md#integer-literals

https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/lexical_conventions/string_literals.md#escape-sequences

Reply:
Ok cool, thanks for clarifying - will revert!